packages/table/src/common/_loading.scss

```diff
@@ -14,25 +14,29 @@
   }
 }
 
-.bp-table-cell,
-.bp-table-header {
-  &.pt-loading {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    cursor: default;
+@mixin hide-text() {
+  // Always set color to transparent so cell content can fade-in when transitioning from the
+  // loading state
+  // stylelint-disable-next-line declaration-no-important
+  color: transparent !important;
+}
 
-    // Always set color to transparent so cell content can fade-in when transitioning from the
-    // loading state
-    // stylelint-disable-next-line declaration-no-important
-    color: transparent !important;
-    pointer-events: none;
-  }
+@mixin disable-interaction() {
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin vertical-center() {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

Comment:
Ok, it can stay here then if it's specific to loading. The name still doesn't work -- not clear that's related to loading. What about `skeleton-vertical-center`?